adds news post for schema sandbox and ion-schema-rust v0.5.0 release (#85)
amzn/ion-schema: _posts/2022-09-12-web-schema-sandbox.md

---
layout: news_item

> title: "Web Schema Sandbox"

Reviewer: You can put "Web" in there somewhere if you like, but I think we should have "Ion" and "Schema" together.
Also, I think something like "New tool" helps make it more clear/gives context about what "Ion Schema Sandbox" is.
```suggestion
title: "New tool: Ion Schema Sandbox"
```

date: 2022-09-12
categories: news ion-schema
---

> A Web Schema Sandbox was added to this public GitHub website on 12 September 2022.

Reviewer: Suggestion for different wording because
- Date is already in the metadata of the article.
- "GitHub website" may be confusing/misleading to some people
- "Ion Schema" rather than "Web Schema"

```suggestion
A browser-based sandbox environment for Ion Schema has been added to the Ion Schema website.
```

This sandbox can be used to validate Ion values for a particular type defined in Ion Schema. 

> The sandbox is created using `ion-schema-rust` latest pre-release version.

Reviewer: I thought it used `0.4.0`, but `ion-schema-rust 0.5.0` was just released. Is that correct?

Author: Yes, but it does use `0.5.0 `as the wasm that was generated used `ion-schem-rust 0.5.0`

Reviewer: Cool!

> | [Web Schema Sandbox]({{site.baseurl}}/sandbox) |

Reviewer: ```suggestion
| [Ion Schema Sandbox]({{site.baseurl}}/sandbox) |
```
